articles/iot-dps/how-to-troubleshoot-dps.md

```diff
@@ -62,8 +62,11 @@ Use this table to understand and resolve common errors.
 |-------|------------|------------|
 | 400 | The body of the request is not valid; for example, it cannot be parsed, or the object cannot be validated.| 400 Bad format |
 | 401 | The authorization token cannot be validated; for example, it is expired or does not apply to the request’s URI. This error code is also returned to devices as part of the TPM attestation flow. | 401 Unauthorized|
-| 404 | The Device Provisioning Service instance, or a resource (e.g. an enrollment) does not exist. |404 Not Found |
+| 404 | The Device Provisioning Service instance, or a resource (e.g. an enrollment) does not exist. | 404 Not Found|
+| 405 | The client service knows the request method, but the target service doesn't recognize this method; for example, a rest operations is missing the enrollment or registration Id parameters | 405 Method Not Allowed |
+| 409 | The request could not be completed due to a conflict with the current state of the target Device Provisioning Service instance; for example, the customer has already created the data point and is attempting to recreate the same datapoint again. | 409 Conflict |
 | 412 | The ETag in the request does not match the ETag of the existing resource, as per RFC7232. | 412 Precondition failed |
+| 415 | The server refuses to accept the request because the payload format is in an unsupported format. For supported formats, see [Iot Hub Device Provisioning Service REST API](https://docs.microsoft.com/en-us/rest/api/iot-dps/) | 415 Unsupported Media Type |
```

Comment:
```suggestion
| 415 | The server refuses to accept the request because the payload format is in an unsupported format. For supported formats, see [Iot Hub Device Provisioning Service REST API](/rest/api/iot-dps/) | 415 Unsupported Media Type |
```
removing loc from link
making site relative link